.github/workflows/nodejs.yml

```diff
@@ -0,0 +1,35 @@
+name: auth-landing CI
+
+on:
+  push:
+    branches: [master]
+  pull_request:
+    branches: [master]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+
+    strategy:
+      matrix:
+        node-version: [10.x, 12.x]
+
+    steps:
+      - uses: actions/checkout@v2
+      - name: Use Node.js ${{ matrix.node-version }}
+        uses: actions/setup-node@v1
+        with:
+          node-version: ${{ matrix.node-version }}
+      - run: npm ci
+      - name: Format code
+        run: npm run format
```

Comment:
It seems `npm run format` would exit with 0 in most cases, so it's redundant.
Instead, we should make CI fail if the format is not correct.